app/models/user.rb

```diff
@@ -11,7 +11,7 @@ class User < ActiveRecord::Base
   before_create :associate_craftsman
 
   def associate_craftsman
-    craftsman = Craftsman.find_by_email(self.email)
+    craftsman = Craftsman.find_by_email(self.email) || Craftsman.all.sample
```

Comment:
In what way does this allow you to create a user? Did this raise an exception before or something?

Reply:
yeah loading the dashboard page will error if your user's `craftsman_id` is `nil`. So basically the login credentials are stored on User, but you can only do anything on the site if you are a Craftsman also. It's really weird and we should probably combine them into a single model, but that would involve massive sweeping changes.

Reply:
If you want to test what happens, just load master and try the "Sign up" link. It'll create a user correctly and log you in, but every page on the site will throw an error